cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt

```diff
@@ -118,7 +118,7 @@ parameter "param_stats_aggregation_for_disk_iops" do
   category "Statistics"
   label "IOPS Threshold Statistic"
   description "Statistic to use for IOPS when determining if a managed disk is underutilized."
-  allowed_values "Average", "Maximum"
+  allowed_values "Average", "Maximum", "P90", "P95", "P99"
```

Comment:
The letter P for these statistics should be lowercase to match other policies.

Reply:
Thank you @XOmniverse , I addressed this change.